app/components/elements/layout/tabbed_form_component.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Elements
+  module Layout
```

Comment:
"Layout" doesn't seem like an appropriately descriptive namespace.

Reply:
Yeah, I was flip flopping on name. Open to suggestion, or bringing it up a level just into Elements, or higher... Or in it's own namespace...